dev/app-shell/elements/arc-view.html

```diff
@@ -10,15 +10,15 @@
 <script>
   class ArcView extends XenBase {
     static get observedAttributes() { return ['arc', 'options', 'data']; }
-    _update(props, state, lastProps) {
+    async _update(props, state, lastProps) {
```

Comment:
This is overriding XenState._update and XenState has what looks like synchronous logic that runs update and in same method calls didUpdate subsequently without an await in there. So that seems like it could be problematic. Can we keep this non-async and use await on whatever involved method calls prompted it?

Reply:
state.view can only be populated asynchronously now, because this._createView is async (because arc.createView and manifest.newView are).

We could this._createView().then(...) but really, that doesn't actually change the semantic here. Everything up to the await will still run on invocation of _update; everything after it will still run after _createView is done, and after _update returns.

Reply:
Ah, I see. Let's try it! 👍